.github/workflows/ci.yml

```diff
@@ -84,6 +84,7 @@ jobs:
 
       - run: pnpm install
       - run: pnpm build
-      - run: pnpm ember try:one ${{ matrix.ember-try-scenario}} -- --config-path tests/dummy/config/ember-try.js
+      - run: pnpm ember try:one ${{ matrix.ember-try-scenario}} --skip-cleanup 
+        working-directory: test-app
```

Comment:
I think we should do `pnpm --filter test-app try:one ${{ matrix.ember-try-scenario}}` instead of navigating to that specific directory beforehand.

Reply:
done